src/plugins/ncr/screens/node/AbstractDelegate.js

```diff
@@ -546,7 +547,9 @@ export default class AbstractDelegate {
       if (!document.hidden) {
         const node = this.getNode();
         this.dispatch(sendHeartbeat(`${nodeRef}`, node ? node.get('etag') : null));
-        this.startCollaborationMonitor();
+        if (isEditMode) {
```

Comment:
@gdbrown i think i should have removed this when i reverted the position, eh? since we know isEditMode is true because of `if (!isEditMode || !canCollaborate || !isCollaborating) {`

Reply:
@efjacobson yes, revert all AbstractDelegate changes